ComponentKit/DataSources/CKCollectionViewTransactionalDataSource.mm

```diff
@@ -85,7 +89,7 @@ - (void)transactionalComponentDataSource:(CKTransactionalComponentDataSource *)d
                        byApplyingChanges:(CKTransactionalComponentDataSourceAppliedChanges *)changes
 {
   [_collectionView performBatchUpdates:^{
-    applyChangesToCollectionView(changes, _collectionView);
+    applyChangesToCollectionView(_collectionView, _attachController, [_componentDataSource state], changes);
```

Comment:
It was not necessary to change the order of operations here after all: just pass the current state as one of the new the arguments.